app/iht/utils/ApplicationKickOutHelper.scala

```diff
@@ -16,21 +16,12 @@
 
 package iht.utils
 
-import iht.constants.IhtProperties
```

Comment:
Make the below line shorter as its too long
 def returnLinkUrl(kickoutReason: String, ihtRef: String, applicationLastSection: Option[String], applicationLastID: Option[String]): Option[Call] = {

*Might refactor by having parameters on the seprate line